packages/common-utils/src/sqlFormatter.ts

```diff
@@ -0,0 +1,462 @@
+// custom dialect ref: https://github.com/sql-formatter-org/sql-formatter/blob/master/docs/dialect.md#custom-dialect-configuration-experimental
```

Comment:
Have we checked this dialect is clickhouse compatible? I'd love us to maintain this going forward and contribute to the upstream later :)

Reply:
yeah, I tested brackets query with local CH and it works. However we will need more test case in future to make sure it cover all clickhouse query.